txcache/errors.go

```diff
@@ -19,3 +19,4 @@ var errNonceGap = errors.New("nonce gap")
 var errExceededBalance = errors.New("exceeded balance")
 var errNonceNotSet = errors.New("nonce not set")
 var errReceivedLastNonceNotSet = errors.New("received last nonce not set")
+var errInvalidMaxTrackedBlocks = errors.New("invalid max tracked blocks")
```

Comment:
`var ErrBadMaxTrackedBlocks = errors.New("bad max tracked blocks")` 

vs.

`var errInvalidMaxTrackedBlocks = errors.New("invalid max tracked blocks")`

Maybe use the same error message (nitpicking).